appcues/src/test/java/com/appcues/data/mapper/step/primitives/OptionSelectPrimitiveMapperTest.kt

```diff
@@ -0,0 +1,76 @@
+package com.appcues.data.mapper.step.primitives
+
+import com.appcues.data.remote.response.step.primitive.PrimitiveResponse.OptionSelectPrimitiveResponse
+import com.appcues.data.remote.response.step.primitive.PrimitiveResponse.OptionSelectPrimitiveResponse.OptionItem
+import com.appcues.data.remote.response.step.primitive.PrimitiveResponse.TextPrimitiveResponse
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import java.util.UUID
+
+class OptionSelectPrimitiveMapperTest {
```

Comment:
tests here are for the mapping logic rules specifically
* if `minSelections` > `options.count` - set `minSelection` = `options.count`
* if `maxSelections` < `minSelections` - set `maxSelections` = `minSelections`